src/GrumPHP/EventListener/TaskEventListener.php

```diff
@@ -189,24 +216,24 @@ protected function readTaskConfigFile($type, $file)
      *
      * @param array|null $data The configuration data.
      */
-    protected function writeTaskConfigFile($type, $file, array $data)
+    private function writeTaskConfigFile($type, $file, array $data): void
```

Comment:
Waarom is private nodig?

Reply:
Class is final, no use to make it protected.